extensions/wikia/AdEngine/js/provider/taboola.js

```diff
@@ -16,7 +16,20 @@ define('ext.wikia.adEngine.provider.taboola', [
 		readMoreDiv = document.getElementById('RelatedPagesModuleWrapper'),
 		context = adContext.getContext(),
 		isMobile = context.targeting.skin === 'wikiamobile',
-		pageType = context.targeting.pageType;
+		pageType = context.targeting.pageType,
+		recirculationWikis = {
```

Comment:
Thanks for checking! I didn't push it to CR because I wanted to check it and happy-path test it ;)

Good idea, will do! :)